sql-tests/testdata/vacuum_engine.sql

```diff
@@ -44,7 +44,7 @@ select isnt_empty(
 
 -- there ought not be any results from the view because no chunks are compressed
 select is_empty(
-    'select * from _ps_catalog.chunks_to_freeze',
+    'select * from _ps_catalog.compressed_chunks_to_freeze',
```

Comment:
I'd add some tests here to query the view even if you expect 0 results. Its probably hard to get a reproduction with non-0 results and probably not worth the effort

Reply:
will do